application/controllers/service_helper.py

```diff
@@ -112,3 +112,25 @@ def service_request(service, endpoint, search_value):
     response = requests.get(service_url, auth=auth)
     response.raise_for_status()
     return response
+
+
+def collection_instrument_link(json_message):
+    """
+    Makes a post request to collection exercise service acknowledging collection instrument load
+    :param: json_message
+    :type: json
+    :return: response
+    """
+
+    auth = (current_app.config.get('SECURITY_USER_NAME'), current_app.config.get('SECURITY_USER_PASSWORD'))
+
+    try:
+        collection_exercise_url = current_app.config['COLLECTION_EXERCISE_URL']
+        url = f'{collection_exercise_url}/collection-instrument/link'
+        log.info('Making request to collection exercise to acknowledge instrument load')
+    except KeyError:
+        raise RasError("collection exercise service not configured", 500)
+
+    response = requests.post(url, json=json_message, auth=auth)
+    response.raise_for_status()
```

Comment:
What happens with the exceptions that this may throw? 

If we don't need to deal with them, do we need to call raise for status here?

Reply:
I think yes, we want to record that the communication between two micro service was not successful isn't? It should be recorded both places.. collex and collex instrument?